game/yodesk.ksy

```diff
@@ -254,7 +254,7 @@ types:
         type: u4
       - id: shared_counter
         type: u2
-        # contents: 0xFF FF
+        valid: 0xFF FF
```

Comment:
Is it really recognized as a number?

Reply:
> Is it really recognized as a number?

Two answers:

1. in the badly-written YAML parser (i.e. https://github.com/jeremyfa/yaml.js) used in the [Web IDE](https://ide.kaitai.io/devel/): **yes**,
2. using a specification-compliant parser like https://yaml-online-parser.appspot.com/: **no** (`valid: 0xFF FF` will be interpreted as `{ "valid": "0xFF FF" }`, and this will fail in the expression parser with an exception `parsing expression '0xFF FF' failed on 1:6`).

And yeah, we should really at least switch to a better parser in the Web IDE.

Reply:
The fix would be

```suggestion
        valid: 0xFF_FF
```

Reply:
Good catch! I was indeed using the web IDE to test these changes (and loved it btw). Too bad its parser is off.